<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0">
        <widget class="QLineEdit" name="flash_input_box"/>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="flash_browse_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="hdd_input_box"/>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="hdd_browse_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLineEdit" name="xiso_input_box">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="flash_label">
         <property name="text">
          <string>Flash ROM path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="bin_browse_button">
         <property name="toolTip">
          <string>Browse for XQEMU bin</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="bin_input_box">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="xiso_label">
         <property name="text">
          <string>XISOs path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="mcpx_label">
         <property name="text">
          <string>MCPX ROM path</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="hdd_label">
         <property name="text">
          <string>HDD path</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="xiso_browse_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="bin_label">
         <property name="text">
          <string>XQEMU bin path</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="mcpx_input_box"/>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="mcpx_browse_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Flags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="expand_ram">
         <property name="text">
          <string>128 MB RAM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="full_boot_anim">
         <property name="text">
          <string>Show full boot animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hdd_unlocked">
         <property name="text">
          <string>Unlocked HDD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enable_kvm">
         <property name="text">
          <string>Enable CPU Acceleration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enable_gl">
         <property name="text">
          <string>Enable SDL OpenGL (WARNING: Only available in perf-wip branch)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <widget class="QLabel" name="controller_label_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Controller 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="controller_label_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Controller 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="controller_label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Controller 3</string>
       </property>
      </widget>
      <widget class="QLabel" name="controller_label_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Controller 4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="controller_bool_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>105</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Plugged in</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="controller_bool_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>105</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Plugged in</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="controller_bool_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>105</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Plugged in</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="controller_bool_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>105</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Plugged in</string>
       </property>
      </widget>
      <widget class="QComboBox" name="controller_select_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keyboard</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="controller_select_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keyboard</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="controller_select_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keyboard</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="controller_select_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keyboard</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="addNetworkRule_button">
         <property name="text">
          <string>Add rule</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="remNetworkRule_button">
         <property name="text">
          <string>Delete rule</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="resNetworkRule_button">
         <property name="text">
          <string>Restore defaults</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QTableView" name="netRules"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current commandline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="currentArguments">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
          </font>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Extra arguments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="extraArguments">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
          </font>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Put extra arguments here.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
